cellpack/autopack/Analysis.py

```diff
@@ -2215,17 +2215,15 @@ def pack_one_seed(
         """
         Packs one seed of a recipe and returns the recipe object
         """
-        seed_basename = f"seed_{seed_index}_{packing_basename}"
-        self.env.result_file = str(
-            self.env.out_folder / f"results_seed_{seed_index}_{packing_basename}"
-        )
+        seed = seed_list[seed_index]  # int(time())
+        seed_basename = self.env.add_seed_number_to_base_name(seed)
```

Comment:
We may not even need lines `2219` and `2220` since `self.pack` calls `self.env.pack_grid` which sets the result file name anyway. Can this be verified?